deploy/002-interest-rate-model.ts

```diff
@@ -22,6 +22,14 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     autoMine: true,
     args: [0, "40000000000000000", "1080000000000000000", "700000000000000000"],
   });
+
+  await deploy("InterestRateModelVBNB", {
```

Comment:
Should we add `if (network.name === "bscmainnet")` for BNB chain specific interest rate models?

Reply:
We have defined `func.skip` (at the bottom of this file), theoretically limiting the deployment to `hardhat`. It seems incoherent and wrong